open_rfps/Decentralized Forum RFP

```diff
@@ -0,0 +1,65 @@
+**Proposal Due Data**
+20th January, 2022
+
+**Proposal Overview**
+Creating a new decentralized forum that will have XOR as the native currency
+
+**Proposal Goals (list with explanations where possible)**
+The main goal is to create a new decentralized forum that will be used by many people and will also bring new users to the Sora network. 
+The forum won't require registration by using email or any kind of KYC, people will be able to post threads and comments 
+by only signing messages with their wallets (polkadot.js). There are various other crypto-forums/platforms (i.e. bitcointalk) that have many million users. 
+Only last year over 20 million users (combined) registered to these forums. 
+There will be various sections in the forum but the very interesting one will be the Marketplace where people will freely trade, 
+buy and sell digital and non-digital goods using a contract system(at their own risk).
+The XOR will be used for all the payments in the forum (like subscriptions, contracts, paid stickers, etc).
+To run the forum, me and my team will have to register a company, so we can pay all the people who will work for the project legally (coders, UI designers, mods, etc.), pay for
+the servers that will host the forum and also probably we will use some of the funds for marketing. I will write the proposed budget below.
+
+**Scope of Work (describe the scope of work in as much detail as possible)**
+All the technology that we may need to code the forum is available. Polkadot.js wallet seems to be compatible with whatever we have in our mind as team. 
+Would be great if we could use wallet connect for Fearless Wallet, Sora DAE and other mobile wallets. This may happen in the future.
+
+**Current roadblocks**
+None 
+
+**Evaluation metrics and criteria**
+- People will be able to post comments and threads by only signing messages with their wallets.
+
+- There will be sections such as Economy, Life, Code, Games, Tech, Science, Market, Cryptocurrencies, and many others.
+
+- To prevent spam posts/threads, it will be required each user to hold at least 40-50$ worth of XOR in their address (the equivalent amount of $ may change).
+
+- There will be an exclusive Sora and Polkaswap section in the forum so all the users can read about new updates and learn more about the network.
+
+- We will create a contract system for the Marketplace section, people will give a score for the quality of the products/services of the vendors after every deal.
```

Comment:
What contract do you mean? How the system will track the contract execution?

Reply:
There will be a contract system that will work on-site only. All the people on the forum will use it in order to have deals with other members. As they start the contract they will explain in detail what is the contract about and also what are the terms of the deal. Once the deal is done, they will both rate each other customer-vendor. Also, they will have the ability to open a dispute in public and explain what went wrong so other members will deal with caution in case something shady happened with that customer/or vendor.

Reply:
There will be a contract system that will work on-site only. All the people on the forum will use it in order to have deals with other members. As they start the contract they will explain in detail what is the contract about and also what are the terms of the deal. Once the deal is done, they will both rate each other customer-vendor. Also, they will have the ability to open a dispute in public and explain what went wrong so other members will deal with caution in case something shady happened with that customer/or vendor.